src/antsibull_changelog/copier.py

```diff
@@ -0,0 +1,129 @@
+# Author: Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2020, Ansible Project
+
+"""
+Directory and collection copying helpers.
+"""
+
+from __future__ import annotations
+
+import os
+import shutil
+import subprocess
+import tempfile
+
+from .config import ChangelogConfig, PathsConfig
+from .logger import LOGGER
+from .utils import detect_vcs
+
+
+class Copier:  # pylint: disable=too-few-public-methods
```

Comment:
Why not make `Copier.copy` and `GitCopier.copy` regular functions?

Reply:
This way it's easier to make them configurable at some point, like allowing to pass the path to `git` to `GitCopier.__init__`, or some other configuration. This obviously would also work by creating a function which creates a function, but classes are cleaner for that IMO.

Reply:
Okay, if you have future plans, I guess, but having a class with one method, no constructor, and no instance variables seemed off.